package-lock.json

```diff
@@ -0,0 +1,6168 @@
+{
```

Comment:
Ehh, I'm not a big fan of leaving `package-lock.json` in open source repos. In the best case, it locks down floating dependencies and people can't get security updates. In the worst case, it lulls developers on the repo into a false sense that everything is fine and up to date =/

Reply:
I'm glad to self-edit this PR to remove it but want to make sure I'm understanding why you added it. Can you explain further?

Reply:
I added it for its normal use of facilitating reproducible builds.
However, this is your repo so since you don't want it I'll simply remove it from my PR - no problem at all.

Reply:
Done. 👍 
Good to go now?

Reply:
Ah, gotcha. Yep, LGTM =)